css/main.css

```diff
@@ -0,0 +1,1109 @@
+html {
+    padding: 0;
+    margin: 0;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+body {
+    display: flex;
+    min-height: 100px;
+    background-color: #000;
+    margin: 0;
+    padding: 0;
+}
+
+.logo {
+    height: 20px;
+    width: 20px;
+}
+
+.logo img {
+    position: absolute;
+    height: 70px;
+    width: 70px;
+    left: 40px;
+    top: 18px;
+}
+
+.nav {
+    display: flex;
+    align-items: flex-end;
+    position: absolute;
+    top: 48px;
+}
+
+.nav_right-20px {
+    right: 300px;
+}
+
+.nav__list {
+    margin: 0;
+    padding: 0;
+    display: block;
+    list-style-type: none;
+    top: 48px;
+    font-size: 16px;
+    line-height: 24px;
+}
+.nav__list li > a {
+    color: #FFF;
+    text-decoration: none;
+}
+
+.nav__list li {
+    float: left;
+    margin: 0 50px 0 0;
+}
+
+.nav__list li:last-child {
+    margin-right: 0;
+}
+
+
+ .login > button {
+    display: block;
+     position: absolute;
+    background: linear-gradient(94.26deg, #C89AFC 9.51%, #7C6AFA 90.23%);
+    border-radius: 50px;
+    width: 183px;
+    height: 44px;
+    font-size: 16px;
+    text-align: center;
+    float: right;
+    right: 30px;
+    top: 48px;
+    justify-content: center;
```

Comment:
It's unused. Because it works with `display: flex;` only. And above you used `float: right;`.